test/tokenomics/rootGaugeLocal.ts

```diff
@@ -255,4 +255,70 @@ describe("Root Gauge (Local)", () => {
       expect(await rootGauge.name()).to.eq(expectedName)
     })
   })
+
+  describe("childChainStreamer", () => {
+    beforeEach(async () => {
+      // Increase time stamp by 12 hours, to be well into the current period
+      await increaseTimestamp(DAY/2)
```

Comment:
I've found increaseTimestamp produce inconsistent time skips for some reason.
Better to use `setTimestamp(current timestamp + delta)` for now